jsonschema2pojo-cli/pom.xml

```diff
@@ -72,6 +69,7 @@
                         </goals>
                         <configuration>
                             <includeScope>runtime</includeScope>
+                            <outputDirectory>${project.build.directory}/lib</outputDirectory>
```

Comment:
What's the reason for this change?

Reply:
Just condensing the two configuration sections together.  There is only one execution for this plugin so just moved all the configuration to a single configuration block.